install.sh

```
@@ -0,0 +1,424 @@
+NC='\033[0m' # No Color
+
+# Default installation directory
+INSTALL_DIR="/usr/local/bin"
+MAN_DIR="/usr/local/share/man/man1"
+REPO_URL="https://raw.githubusercontent.com/semperai/git-commitai/refs/heads/master"
+
+# Function to print colored output
+print_color() {
+    color=$1
+    shift
+    echo -e "${color}$@${NC}"
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)